frontend/src/lib/components/EntityFilterInfo.tsx

```diff
@@ -11,22 +10,18 @@ interface EntityFilterInfoProps {
     style?: React.CSSProperties
 }
 
-function TextWrapper(props: TextProps): JSX.Element {
-    return (
-        <Typography.Text style={{ maxWidth: 400 }} {...props}>
-            {props.children}
-        </Typography.Text>
-    )
-}
-
 export function EntityFilterInfo({
     filter,
     allowWrap = false,
     showSingleName = false,
     style,
 }: EntityFilterInfoProps): JSX.Element {
     if (isAllEventsEntityFilter(filter) && !filter?.custom_name) {
-        return <TextWrapper title="All events">All events</TextWrapper>
+        return (
+            <div className="text-primary font-medium whitespace-nowrap max-w-100" title="All events">
```

Comment:
Looking at visual snapshot changes, this shouldn't have `text-primary font-medium`. The original `TextWrapper` didn't include those classes